README.md

```diff
@@ -76,6 +76,22 @@ $ poetry install --no-root
 
 The Pantos Common project should be used as a utility library, for example as a submodule in an upstream project. After those steps, the modules can be imported directly from the Common library.
 
-### 3.1 Examples
+### 3.1 Configuration
+
+The Pantos Common library allows its configuration to be loaded multiple fold. This normally involves an environment file and a base YAML configuration file, which can be located in the following predefined paths:
+
+```
+    cwd
```

Comment:
`$PWD` and `$HOME` are usually used in paths for the working and home directory, respectively.

Reply:
We can also use ~ to indicate home

Reply:
I'm fine with either.